devops/Dockerfile

```diff
@@ -0,0 +1,33 @@
+﻿# Используем официальный образ .NET SDK для сборки приложения
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+
+# Устанавливаем рабочую директорию
+WORKDIR /app
+
+# Копируем csproj и восстанавливаем зависимости
+COPY Memo4me.sln ./
+COPY src/Core/Memo4me.Application/*.csproj ./src/Core/Memo4me.Application/
+COPY src/Core/Memo4me.Domain/*.csproj ./src/Core/Memo4me.Domain/
+COPY src/Infrastructure/Memo4me.Persistence/*.csproj ./src/Infrastructure/Memo4me.Persistence/
+COPY src/Presentation/Memo4me.API/*.csproj ./src/Presentation/Memo4me.API/
+COPY src/Presentation/Memo4me.UI/*.csproj ./src/Presentation/Memo4me.UI/
```

Comment:
I don't see any usages for this UI projects, and I believe you don't have references on it